src/lib/itemFormUtils.ts

```diff
@@ -47,4 +49,24 @@ export function toTitleCase(str: string): string {
         titleCase += capitalizedWord + " ";
     });
     return titleCase.trim();
+}
+
+export async function uploadImageForItem(item: Item, data: FormData) {
+    var file: File = data.get("image") as File;
```

Comment:
why var not const? Really don't want to block the PR on this, bt did you have a reason

Reply:
Oversight, will change